cd/python/docker/Dockerfile

```diff
@@ -36,6 +36,12 @@ RUN apt-get install -y libgomp1
 
 ARG MXNET_VARIANT
 RUN if [ "$MXNET_VARIANT" = "aarch64_cpu" ] ; then echo "not installing libquadmath0 on aarch64" ; else apt-get install -y libquadmath0 ; fi
+RUN if [ "$MXNET_VARIANT" = "aarch64_cpu" ] ; \
+    then wget https://armkeil.blob.core.windows.net/developer/Files/downloads/hpc/arm-performance-libraries/21-0-0/Ubuntu18.04/arm-performance-libraries_21.0_Ubuntu-18.04_gcc-10.2.tar && \
+    tar -xvf arm-performance-libraries_21.0_Ubuntu-18.04_gcc-10.2.tar && \
+    arm-performance-libraries_21.0_Ubuntu-18.04_gcc-10.2/arm-performance-libraries_21.0_Ubuntu-18.04.sh -a; \
+    fi
```

Comment:
missing `else apt-get install -y libquadmath0`

Reply:
It's in the line above. I thought I would use two `RUN`'s because libqudamath and apl are two different packages, what would you think?

Reply:
Oh I see, missed that